sdk/js/src/service/gateways.js

```diff
@@ -184,6 +187,41 @@ class Gateways {
     // Combine all stream sources to one subscription generator.
     return combineStreams(streams)
   }
+
+  // Gateway Configuration Server.
+
+  async getGlobalConf(gatewayId) {
```

Comment:
I still think we shouldnt add this to the sdk, at least with this implementation.
Another solution would be to make a sepate function in `console/api/index.js` to fetch the global conf.
We can also try to find a way to reuse functionality from the `http` connector in `getGlobalConf` method or any other methods we might add in the future that are not in the grpc api.

Reply:
The fact that it's not part of the RPC API makes it a bit of a tinker indeed but I think functionality-wise, the SDK is the right place to have this, seeing that people with other SDK use-cases than ours might want to access this functionality as well. I agree with @benolayinka that we can change the implementation to be more native once this is added to the RPC API. This will be easier and less disruptive when it is already here than if we would port it from the console.

Is there any pressing concern I am missing?

Reply:
Please provide an api for registering methods that are not defined in the rpc api.

My only concern is current implementation.

Reply:
Just to clarify. We already have the http connector in every service. So the simplest solution would be to reuse token extraction and error handling functionality from the connector in the gateways service.
Im also fine with adding a `TODO` comment here referencing issue.

Reply:
Using the connector now directly. I previously decided against this since it necessitates to access the "private" \`\_connector\` method of the API class. But I guess in this special case it's fine to do that until we resolve this with #3280